contact.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+
+		<title>Sprint Challenge - Contact</title>
+		<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
+		<link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+		<link rel="stylesheet" href="css/index.css">
+	</head>
+	<body>
+		<header>
+		<div id="navbar" class="blog-nav">
+		<img  class="logo" src="img/lambda-black.png" alt="Lambda Company Logo">
+		<nav>
+		<a href="index.html" class="nav">Home</a>
+		<a href="about.html" class="nav">About</a>
+		<a href="#" class="nav">Products</a>
+		<a href="blog.html" class="nav">Blog</a>
+		<a href="contact.html" class="nav">Contact</a>
+		</nav>
+		</div>
+		</header>
+	
+		<div class="container">
+
+			<img src="img/jumbo.jpg" alt="header image" id="top-image">
+
+			<div id="intro">
+				<h1>Questions or comments? We'd love to hear from you!</h1>
+				<h1>Fill out the form below or connect with us on social media.</h1>
+			</div>
+
+			<div id="contact-main">
+				
+				<form action="index.html">
+
+					<fieldset>
+					<label for="name">Name</label>
+					<input type="text" name="fullname" id="name" placeholder="Your full name..." required="required">
+
+					<label for="email">Email</label>
+					<input type="text" name="email" id="email" placeholder="you@youremail.com" required="required">
+					<label for="address">Address</label>
+					<input type="text" id="address1" placeholder="Address line 1..." required="required">
+					<input type="text" id="address2" placeholder="Address line 2...">
+					<div id="city-st-zip">	
+					<div id="city">
+					<label for="city">City</label>
+					<input type="text" name="city" placeholder="Enter your city..." required="required">
+					</div>
+
+					<div id="zip">
+					<label for="zip">Zip Code</label>
+					<input type="text" name="zipcode" placeholder="XXXXX" maxlength="5" required="required">
+					</div>
+					<div id="state">
+					<label for="state">State</label>
```

Comment:
Woah you added all 50 states? We didn't go over drop down options so great job added that!